Gemfile

```diff
@@ -9,6 +9,7 @@ gem 'pg', '>= 0.18', '< 2.0'
 gem 'puma', '~> 3.7'
 gem 'rack-cors', require: 'rack/cors'
 gem 'active_model_serializers', '~> 0.10.0'
+gem 'will_paginate', '~> 3.1.0'
```

Comment:
I can't really see that you actually need this gem in this iteration of this functionality. The "problem" is that now that you have added it, I will need to see some tests for fetching the consequent pages. That is unnecessary complexity and a bit of premature optimization. I would have saved this for a later story if I ware you.

Reply:
As discussed we will proceed with pagination at this stage but we see your point. Metadata added to paginated response, with request specs.